### PR TITLE
ICS no-wallet error presentation

### PR DESCRIPTION
**Pull request by mlsteele**
image

### CODE REVIEW

File: shared/chat/payments/confirm/container.js [hunk 1]
```diff
@@ -1,7 +1,11 @@
 // @flow
 import * as Chat2Gen from '../../../actions/chat2-gen'
-import {namedConnect} from '../../../util/container'
+import * as Container from '../../../util/container'
+import * as RouteTreeGen from '../../../actions/route-tree-gen'
+import * as SettingsTabs from '../../../constants/settings'
+import * as Tabs from '../../../constants/tabs'
 import PaymentsConfirm from '.'
+import {namedConnect} from '../../../util/container'
```
Comment (buoyad) [inline shared/chat/payments/confirm/container.js]: :fire: dupe import, use `Container.namedConnect` below

**Comment by mlsteele** [comment]
updated
nowallet
image
error

File: shared/chat/payments/confirm/index.js [hunk 2]
```diff
@@ -167,6 +206,9 @@ const styles = Styles.styleSheetCreate({
       padding: 20,
     },
   }),
+  happyErrorText: {
```
Comment (buoyad) [inline shared/chat/payments/confirm/index.js]: don't need this style, it's this color by default

File: shared/chat/payments/confirm/index.js [hunk 3]
```diff
@@ -27,15 +38,41 @@ const _PaymentsConfirmError = (props: ErrorProps) => (
...
+        </Kb.Text>
+      </Kb.Box2>
+      <Kb.Box2 direction="vertical" style={styles.pushDown} fullWidth={true}>
+        <Kb.ButtonBar align="center" direction="row" fullWidth={true} style={styles.buttonBar}>
+          <Kb.Button type="Secondary" onClick={props.onCancel} style={styles.cancelButton} label="Cancel" />
+          <Kb.Button
+            style={styles.submitButton}
+            type="Primary"
+            onClick={props.onWallet}
+            label={'Set up wallet'}
```
Comment (buoyad) [inline shared/chat/payments/confirm/index.js]: `{'...'}` -> `"..."`

**Pull request merged**